bittensor/_receptor/receptor_pool_impl.py

```diff
@@ -133,28 +191,109 @@ def forward (
         # Init function.
         def call_forward( args ):
             return args['receptor'].forward( args['synapses'], args['inputs'], args['timeout'] )
+        
+        # Unpack responses
+        results_dict = dict(
+            outputs=[],
+            codes=[],
+            times=[],
+            uids=[]
+        )
+
+
+        # resolve the minumum number of successes
+        if min_success == None:
+            min_success = len(endpoints)
+        if min_success < 1:
+            min_success = int(min_success*len(endpoints))
+        elif min_success >= 1:
+            min_success = int(min(min_success,len(endpoints)))
+        elif min_success <= 0:
+            raise Exception(' REQUIRED: 0<min_success<len(endpoints)')
+
+
+        # ensure max_workers
+        if not isinstance(max_workers, int):
+            max_workers = len(endpoints)
```

Comment:
will we run into any performance issues if we try to query too many endpoints? would a factor of `multiprocessing.cpu_count()` be safer?

Reply:
This is for threading only. I think we should assume 1 process per receptor pool. This is also the default to what is already in the receptor pool

Reply:
Oh, I realize I submitted a fix for this myself a while ago. There's actually an unused arg in `init`, `self.max_worker_threads`. Could we use that instead and remove it as an arg from the forward call?